sharding/client/utils.go

```diff
@@ -0,0 +1,39 @@
+package client
+
+import (
+	"fmt"
+)
+
+type collationbody []byte
+
+var (
+	collationsizelimit = int64(2 * *20)
+	chunkSize          = int64(32)
+	indicatorSize      = int64(1)
+	chunkDataSize      = chunkSize - indicatorSize
+)
+
+func (c *collationbody) validateBody() bool {
+	return len(c) < 2^20 && len(c) > 0
+}
+
+func createChunks(c collationbody) {
```

Comment:
I'd rather have a different name than c for this variable. It reads nicer if we call the  variable collationBody or something like that as I can understand line 28 immediately upon the first read.

Reply:
It is go convention to prefer shortest variable name for the given context. 
If anything, we could extend this name to `cb` and still understand that it is the collation body.